Lagerregal/urls.py

```diff
@@ -27,6 +27,7 @@
 import permission; permission.autodiscover()
 from permission.decorators import permission_required
 from django.views.decorators.clickjacking import xframe_options_exempt
+import debug_toolbar
```

Comment:
can we move this into the `if settings.DEBUG:` block?

Reply:
done